test/e2e/scenario/logs.scenario.ts

```diff
@@ -97,8 +98,20 @@ describe('logs', () => {
       expect(serverEvents.logs.length).toBe(1)
       expect(serverEvents.logs[0].message).toBe(`Fetch error GET ${baseUrl}/throw-large-response`)
       expect(serverEvents.logs[0].error?.origin).toBe('network')
-      expect(serverEvents.logs[0].error?.stack?.length).toBeLessThan(LARGE_RESPONSE_MIN_BYTE_SIZE)
+
+      const ellipsisSize = 3
+      expect(serverEvents.logs[0].error?.stack?.length).toBe(DEFAULT_REQUEST_ERROR_RESPONSE_LENGTH_LIMIT + ellipsisSize)
+
       expect(servers.base.app.getLargeResponseWroteSize()).toBeLessThan(LARGE_RESPONSE_MIN_BYTE_SIZE)
+      expect(servers.base.app.getLargeResponseWroteSize()).toBeLessThan(
+        // When reading the request, chunks length are probably not aligning perfectly with the
+        // response length limit, so it sends few more bytes than necessary. Add a 10% margin of
+        // error to verify that it's still close to the expected limit.
+        DEFAULT_REQUEST_ERROR_RESPONSE_LENGTH_LIMIT * 1.1
+      )
+      expect(servers.base.app.getLargeResponseWroteSize()).toBeGreaterThanOrEqual(
+        DEFAULT_REQUEST_ERROR_RESPONSE_LENGTH_LIMIT
+      )
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** what about removing `LARGE_RESPONSE_MIN_BYTE_SIZE` expectation (and then export)?
checks with `‏DEFAULT_REQUEST_ERROR_RESPONSE_LENGTH_LIMIT` seems enough